Source/MLX/Documentation.docc/examples.md

```diff
@@ -5,13 +5,15 @@ Swift example code for MLX and MLXNN.
 @Metadata {
     @CallToAction(
         purpose: download,
-        url: "https://ml-explore.github.io/mlx-swift/mlx-examples-swift.zip")
+        url: "https://github.com/ml-explore/mlx-swift-examples/archive/refs/heads/main.zip")
```

Comment:
We can get rid of our local zip (it lives only in gh-pages so will be dealt with there)